### DISCUSSION TITLE
Sending message to campfire

### DISCUSSION THREAD

**Discussion by shairyar** [excid3/noticed#454] [Q&A] [answered]
I am trying to send the message to the Campfire, and using `deliver_by :discord` seems to do the job 👍🏼

```ruby
class ActivityNotifier < ApplicationNotifier
  bulk_deliver_by :discord do |config|

    config.url = "CAMPFIRE-CHAT-URL-GOES-HERE"
    config.json = -> {
      {
        title: "This is title",
        body: "This is a json body",
        message: "This is a json message",
        text: "This is a json text"
      }
    }
  end
end
```

I am sending the message using the following snippet.

```ruby
ActivityNotifier.with(account: account).deliver
```

The only problem I have is that I see the `JSON` posted as a message. 

<img width="881" alt="image" src="https://github.com/excid3/noticed/assets/5089055/afc00470-9b2b-4835-a7c4-e2ac676a58b7">

Is there a way to send the actual message to Campfire as well? For example, the message posted to Campfire should be "Hey, check this out," followed by the JSON object.

I tried `config.title`,`config.message`, and `config,text`, and nothing seems to work.

**Comment by shairyar** [comment, marked as answer]
So I had a look at the source code of the campfire, and it seems it is working as intended. Whatever you send to campfire gets saved as a message text.

**Comment by lundie** [comment]
@shairyar I added the ability to set the body directly when using webhooks. This would help with pushing messages to campfire. https://github.com/excid3/noticed/pull/555.
